node/dev/Dockerfile

```diff
@@ -7,3 +7,4 @@ FROM node:alpine
 #   && apk add git build-base nodejs-dev@edge nodejs@edge \
 #   && npm install -g npm \
 #   && rm -rf /var/cache/apk/*
+
```

Comment:
Is this as accidental change? If so can we tidy it up? Just so it is super clear what we are changing here.

Reply:
Maybe even clear out the comment in the file if they are no longer needed?

Reply:
yup was accidental - updated